include/eve/traits/overload/supports.hpp

```diff
@@ -63,6 +63,13 @@ namespace eve
       auto    dropped =  rbr::drop(kws, *this);
       return  kumi::tuple{value, options<decltype(dropped)>{dropped}};
     }
+
+    template <rbr::concepts::settings S>
+    constexpr EVE_FORCEINLINE auto operator&&(options<S> const& opt) const noexcept
+    {
+      auto new_opts = rbr::merge(opt, *this);
+      return options<decltype(new_opts)>{new_opts};
+    }
```

Comment:
I am not wild about this overload. operator&& and other operators should only be overloaded if the meaning is super obvious.

What does this do?